uuid.js

```diff
@@ -11,6 +11,9 @@
   // returns 128-bits of randomness, since that's what's usually required
   var _rng;
 
+  // Allow for MSIE11 msCrypto
+  var _crypto = _global.crypto || _global.msCrypto;
```

Comment:
If it helps, here's a screenshot from IE 11's console on a page in edge mode.
Note the different casing - Crypto vs crypto.  Weird / unexpected IMHO.

![capture](https://cloud.githubusercontent.com/assets/1314477/6724861/61bb1be0-ce4b-11e4-9ee7-e0f2162f7e77.PNG)